src/components/ScholarshipDetailCardCell.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles, Grid, Typography } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  container: {
+    color: 'rgba(0, 0, 0, 0.6)',
+    padding: `${theme.spacing(1.5)}px 0`,
+    borderBottom: '1px solid #EEEEEE',
+  },
+  labelWrapper: {
+    '& p': {
+      fontWeight: 'normal',
+    },
+  },
+  textWrapper: {
+    '& p': {
+      textAlign: 'right',
```

Comment:
Instead of this and the fixed grid item sizes of 4 and 8 below I suggest making the grid container `justify` using `space-between`. See the [interactive Grid demo](https://material-ui.com/components/grid/#interactive) for details.